### PR TITLE
fix newsletters query and bump version

### PR DESCRIPTION
**Pull request by isaacmg410**
Fix Newsletters query, in admin site, with same area/department than current user.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This project is in BETA, but going to be tested in production.
 
 ## next version:
 
+## Version 0.0.7 (MINOR)
+- Fix Newsletters query, in admin site, with same area/department than current user.
```
Comment (tramuntanal) [inline CHANGELOG.md]:
```suggestion
- Fix Newsletters query. In admin newsletter list, filter newsletters only with same area/department than current user. 
```

File: app/decorators/decidim/admin/newsletters_controller_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    @collection ||= Decidim::Newsletter.where(organization: current_organization)
+                                       .joins(author: :areas)
+                                       .where("department_admin_areas.decidim_area_id = ?", current_user_areas.pluck(:id))
+
+  end
+
+  def current_user_areas
+    return unless current_user.department_admin?
+    return unless current_user.areas.any?  
+    return current_user.areas if current_user&.department_admin?
```
Comment (tramuntanal) [inline app/decorators/decidim/admin/newsletters_controller_decorator.rb]: Shouldn't this line be enough for this method?
Can't we remove the previous two lines?

Comment (isaacmg410) [reply app/decorators/decidim/admin/newsletters_controller_decorator.rb]: You are right! Change applied!

Comment (tramuntanal) [reply app/decorators/decidim/admin/newsletters_controller_decorator.rb]: I like more the one liner, but well...

**Pull request merged**